console/src/pages/Evaluation/EvaluationListCard.tsx

```diff
@@ -141,27 +141,27 @@ export default function EvaluationListCard() {
 
         if (!summaryTable?.data) return columnsWithAttrs
 
-        Object.entries(summaryTable?.data?.columnTypes ?? {}).forEach(([name, type]) => {
-            if (name === 'id') return
-            switch (type) {
+        summaryTable?.data?.columnTypes?.forEach((column) => {
+            if (column.name === 'id') return
+            switch (column.type) {
                 case 'UNKNOWN':
                 case 'BYTES':
                     break
                 case 'STRING':
                     columnsWithAttrs.push(
                         StringColumn({
-                            key: name,
-                            title: name,
+                            key: column.name,
+                            title: column.name,
                             filterType: 'string',
-                            mapDataToValue: (data: any): string => data.attributes?.[name] ?? '-',
+                            mapDataToValue: (data: any): string => data.attributes?.[column.name] ?? '-',
                         })
                     )
                     break
```

Comment:
Should we complete the switch with new column cases, like LIST/OBJECT, or just ignore?

Reply:
A lot should be changed to support other types. Just ignore these now